### PR TITLE
More projects

### PR DESCRIPTION
**Pull request by LunarWatcher**
Also moves the JS header down under Java, so everything is nice and sorted (purely because I had to fix a conflict, and it was annoying to not fix)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,14 +23,21 @@ Please add new repos in their programming language and keep it alphabetical. [Cl
...
 - [F-Droid Server](https://gitlab.com/fdroid/fdroidserver/) - Tools for managing builds, indexes, updates, and deployments for F-Droid repositories.
 
-## JavaScript
-- [Cloudflare Status Worker](https://github.com/WalshyDev/cf-status-worker) - Get a message in your Discord server when Cloudflare has an issue. Deployed with Clouflare Workers
+## Vimscript
+ - [auto-pairs](https://github.com/LunarWatcher/auto-pairs) - Plugin for automatically inserting and deleting pairs
+ - [Dawn](https://github.com/LunarWatcher/Dawn) - Project template generator
+
+## Zsh
+ - [Oh Ny Zsh](https://github.com/ohmyzsh/ohmyzsh/)
```
Comment (TheLastProject) [inline README.md]: Seems like a typo, should be Oh My Zsh, right?

Comment (TheLastProject) [reply README.md]: Also please add a description

**Review by TheLastProject — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (LunarWatcher) [reply README.md]: aw shit, my bad

**Comment by TheLastProject** [comment]
Thanks!

**Pull request merged**